### PR TITLE
Add annotations property to entity

### PR DESCRIPTION
**Pull request by gregorjerse**
in Python processes. It can be used as dictionary to read/set entity annotations.

### CODE REVIEW

File: resolwe/flow/models/annotations.py [hunk 1]
```diff
@@ -370,6 +370,15 @@ def __str__(self):
 class AnnotationValue(AuditModel):
     """The value of the annotation."""
 
+    class Meta:
```
Comment (dblenkus) [inline resolwe/flow/models/annotations.py:373]: This can/should go into a separate commit.

Comment (gregorjerse) [reply resolwe/flow/models/annotations.py:373]: Done.

**Review by dblenkus — APPROVED** [review]

**Review by dblenkus — APPROVED** [review]
